WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/viewholders/BarChartViewHolder.kt

```diff
@@ -42,6 +46,24 @@ class BarChartViewHolder(parent: ViewGroup) : BlockListItemViewHolder(
         GlobalScope.launch(Dispatchers.Main) {
             delay(50)
             chart.draw(item, labelStart, labelEnd)
+            chart.post {
```

Comment:
I invoked post here because it would initiate the helper when the chart is drawn and that would allow all the virtual views that are needed by the accessibility framework to be created. I am going to check to see if I can use another way to accomplish this that doesn't involve mixing threads which could cause issues or side effects.

Reply:
Thanks ;) - I'd just try the approach I suggested or anything else that comes from the top of your head -> if it doesn't work, don't worry about it.

Reply:
I tried the coroutines approach but it didn't work out well. My coroutine knowledge isn't as advanced so that could probably be the reason so, for now, I think it's okay to leave it once there are no obvious side effects in testing.

Reply:
👍 Thanks for giving it a try! ;)